typing/typemod.ml

```diff
@@ -1384,9 +1382,9 @@ and transl_signature env sg =
             in
             List.iter (fun td ->
               Signature_names.check_type names td.typ_loc td.typ_id;
-              if not (Btype.is_row_name (Ident.name td.typ_id)) then
+              (* if not (Btype.is_row_name (Ident.name td.typ_id)) then
                 Cmt_format.register_uid td.typ_type.type_uid
-                  (Type_declaration td)
+                  (Type_declaration td) *)
```

Comment:
looks like something we want to remove (rather than keep as a comment) once someone in the know confirmed that it is not useful anymore